templates/config.fish

```diff
@@ -6,8 +6,8 @@ set -g OMF_CONFIG {{OMF_CONFIG}}
 
 ### Configuration required to load oh-my-fish ###
 # Note: Only add configurations that are required to be set before oh-my-fish is loaded.
```

Comment:
I feel this would be too intrusive for new users. While I think this is the best, new users may not even be aware of events and simply looking for a regular `config` file to dump their stuff.